setup.py

```diff
@@ -42,7 +40,7 @@ def find_data_files():
     ]
 
 setup_args = dict(
-    name='metaopt',
+    name='metaopt.core',
```

Comment:
Agreed on the need of `metaopt.core` and others, nice to build a community upon ;)